app/models/Channel.scala

```diff
@@ -0,0 +1,20 @@
+package models
+
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto.{deriveEnumerationDecoder, deriveEnumerationEncoder}
+
+sealed trait Channel
```

Comment:
I moved this from `ChannelTests.scala` as I was getting weird circe issues when importing it from there, and the simplest solution is to split the file up